include/wx/webview.h

```diff
@@ -203,6 +203,7 @@ class WXDLLIMPEXP_WEBVIEW wxWebView : public wxControl
     virtual wxWebViewZoom GetZoom() const = 0;
     virtual wxWebViewZoomType GetZoomType() const = 0;
     virtual void SetZoom(wxWebViewZoom zoom) = 0;
+    virtual void SetZoomFactor(float zoom) = 0;
```

Comment:
I've realized that we can now set the zoom factor as `float`, but can still only retrieve it as `wxWebViewZoom`, which is slightly weird.

It won't block merging this PR, but shouldn't we have `float GetZoomFactor() const` too for consistency and completeness?